src/assets/loader/parsers/textures/loadSVG.ts

```diff
@@ -69,7 +69,8 @@ export const loadSvg: LoaderParser<Texture | GraphicsContext, TextureSourceOptio
         loader: Loader
     ): Promise<Texture | GraphicsContext>
     {
-        if (asset.data.parseAsGraphicsContext ?? this.config.parseAsGraphicsContext)
+        asset.data = asset.data ?? this.config;
+        if (asset.data.parseAsGraphicsContext)
```

Comment:
Small suggestion to keep asset immutable.
```suggestion
        const data = asset.data ?? this.config;
        if (data.parseAsGraphicsContext)
```

Reply:
then if would fail in `loadAsTexture()` line 124 same file. For some reason `asset.data` is not set when loading an svg through a spritesheet (if i load the same svg file using `await Asset.load("svgfile.svg")` asset.data is there). I dont think adding this would break anything (it's just meta data).

Reply:
I was trying to add a test case but I am not able to run the test suit for some reason. It just stuck here
![image](https://github.com/pixijs/pixijs/assets/45667865/46f4cfd6-0d2a-4c1d-ab1c-0b55a7fb338a)

I ran `npm run start` to build then ran `npm test`.  I don't know what I am missing any suggestions?

I also tried running the test file i need directly, same thing.
![image](https://github.com/pixijs/pixijs/assets/45667865/39c2b045-416c-4909-8319-d19a6509bf02)

Reply:
> then if would fail in `loadAsTexture()` line 124 same file. For some reason `asset.data` is not set when loading an svg through a spritesheet (if i load the same svg file using `await Asset.load("svgfile.svg")` asset.data is there). I dont think adding this would break anything (it's just meta data).

https://github.com/pixijs/pixijs/blob/54029ce4e39d30db431cd66b8f9161f0880661b9/src/assets/loader/Loader.ts#L111


this line does not add data to asset for some reason

Reply:
> Small suggestion to keep asset immutable.

I tried doing that in both `load` and `loadAsTexture`, it fails later. I think its best to use what I did. I don't think it will cause any issues.

Reply:
If we need to reassign `asset.data` it means there's likely some other problem upstream from this.

Reply:
> Small suggestion to keep asset immutable.

done

Reply:
@bigtimebuddy let me know if this looks good

Reply:
@bigtimebuddy added a test case, but i can't run them so not sure if it's correct (it looks like it should pass)